app/assets/javascripts/discourse/app/lib/link-hashtags.js

```diff
@@ -5,26 +5,17 @@ import { TAG_HASHTAG_POSTFIX } from "discourse/lib/tag-hashtags";
 
 const categoryHashtags = {};
 const tagHashtags = {};
-const checkedHashtags = [];
+const checkedHashtags = new Set();
 
 export function linkSeenHashtags($elem) {
   const $hashtags = $elem.find("span.hashtag");
   if ($hashtags.length === 0) {
     return [];
   }
 
-  const slugs = $hashtags.map((_, hashtag) =>
-    $(hashtag)
-      .text()
-      .substr(1)
-  );
-
-  const unseen = [];
-  _.uniq(slugs).forEach(slug => {
-    if (!checkedHashtags.includes(slug)) {
-      unseen.push(slug);
-    }
-  });
+  const slugs = $hashtags
+    .map((_, hashtag) => hashtag.innerText.substr(1))
+    .toArray();
```

Comment:
Can we do this instead?

```suggestion
  const slugs = [...$hashtags.map((_, hashtag) => hashtag.innerText.substr(1))];
```